package.json

```diff
@@ -62,12 +61,13 @@
     "eslint-config-airbnb": "^19.0.2",
     "html-entities": "^2.1.0",
     "mustache": "^4.1.0",
+    "node-sass": "npm:sass@^1.49.9",
     "react": "^17.0.1",
     "react-dom": "^17.0.1"
   },
   "engines": {
-    "node": "^12.0.0",
-    "npm": "^6.0.0"
+    "node": "^16.0.0",
+    "npm": ">=7.0.0"
```

Comment:
Just as a note, I think this means people working on the docs site need to update their local environments.

Reply:
When using this versions (editing the `.node-version` file) I get this warning: 

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'shortcode-parser@0.0.1',
npm WARN EBADENGINE   required: { node: '0.10.x' },
npm WARN EBADENGINE   current: { node: 'v16.14.2', npm: '8.5.0' }
npm WARN EBADENGINE }
```

I'm not sure this is a problem, I was able to run the site without problems.

Reply:
I was seeing the same thing with shortcode-parser actually but in the last few npm installs I hadn't seen it again. I tried to update the version for it, but it's already on the latest possible.